### DISCUSSION TITLE
VScode Copilot status icon missing in 1.83/1.84

### DISCUSSION THREAD

**Discussion by Abbe98** [community/community#53881] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As of version 1.83+ of the GitHub Copilot VSCode extension the status icon in the bottom-right corner is no longer visible. Downgrading to 1.82 solves the issue.

<details><summary>VSCode version & system details</summary>
<p>

```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:52.732Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 6.2.10-200.fc37.x86_64
Sandboxed: Yes
```

</p>
</details> 

There are a couple of similar issues reported for older versions: #51923 #51901

**Comment by Akash1134** [comment]
Hi @Abbe98 ,

We are aware of the issue and our team is working towards a fix. 

Thanks!
